apache/superset
# feat: Migrates Area chart

### SUMMARY
This PR migrates the Area chart to the ECharts version.

- Removes the legacy Area chart code and unregisters the plugin.
- Removes old plugin references throughout the codebase
- Updates unit tests
- Updates E2E tests
- Updates the example dashboards
- Updates translation files

Note a previous migration existed however the legacy chart wasn't removed and thus new legacy Area charts could have been created.

### AFTER SCREENSHOT
<img width="1085" alt="Screenshot 2023-05-03 at 18 14 16" src="https://user-images.githubusercontent.com/70410625/236051421-eec6f181-c926-43b3-9594-b35e38b9dc50.png">

### TESTING INSTRUCTIONS
1 - Make sure all Area (legacy) charts were converted to the ECharts version
2 - Make sure Area (legacy) is not available anymore in the viz picker
3 - Make sure that it's possible to revert the migration by executing superset db downgrade and reverting this PR

### ADDITIONAL INFORMATION
